wagtail_wordpress_import/test/tests/test_shortcode_handlers.py

```diff
@@ -274,13 +275,75 @@ class FooHandler(BlockShortcodeHandler):
         self.assertEqual(len(SHORTCODE_HANDLERS), 2)
 
 
+class TestProvidedShortcodeIsTopLevel(TestCase):
+    """
+    Block Shortcode Handler has a top_level_property that is True by default.
+    It can be set to False to indicate that the shortcode is not a top level html tag.
+
+    There are tests here to ensure that none boolean values will return True.
+    """
+
+    def test_is_top_level_html_tag_value_false(self):
+        class FooHandler(BlockShortcodeHandler):
+            shortcode_name = "foo"
+            custom_html_tag_prefix = "wagtail_block_"
+            top_level_html_tag = False
+
+        foo = FooHandler()
+        self.assertFalse(foo.is_top_level_html_tag)
+
+    def test_is_top_level_html_tag_value_string(self):
+        class FooHandler(BlockShortcodeHandler):
+            shortcode_name = "foo"
+            custom_html_tag_prefix = "wagtail_block_"
+            top_level_html_tag = "bar"
+
+        foo = FooHandler()
+        self.assertTrue(foo.is_top_level_html_tag)
+
+    def test_is_top_level_html_tag_value_none(self):
+        class FooHandler(BlockShortcodeHandler):
+            shortcode_name = "foo"
+            custom_html_tag_prefix = "wagtail_block_"
+            top_level_html_tag = None
+
+        foo = FooHandler()
+        self.assertTrue(foo.is_top_level_html_tag)
+
+    def test_is_top_level_html_tag_value_list(self):
+        class FooHandler(BlockShortcodeHandler):
+            shortcode_name = "foo"
+            custom_html_tag_prefix = "wagtail_block_"
+            top_level_html_tag = ["foo", "bar"]
+
+        foo = FooHandler()
+        self.assertTrue(foo.is_top_level_html_tag)
+
+    def test_is_top_level_html_tag_value_dict(self):
+        class FooHandler(BlockShortcodeHandler):
+            shortcode_name = "foo"
+            custom_html_tag_prefix = "wagtail_block_"
+            top_level_html_tag = {"foo": "bar"}
+
+        foo = FooHandler()
+        self.assertTrue(foo.is_top_level_html_tag)
```

Comment:
Discussion about this test case was originally started in https://github.com/torchbox/wagtail-wordpress-import/pull/100#discussion_r760992232 (now outdated).

Thanks for adding the docstring. I don't follow why we need to handle the case where this is set as a non-boolean. Some of those cases are very counterintuitive, e.g. why would setting `top_level_html_tag = None` give `obj.is_top_level_html_tag == True`? I would prefer it if setting a non-boolean would be the default truthiness of the object.

I would suggest removing the `is_...` method, and restoring it to a bare attribute (see https://github.com/torchbox/wagtail-wordpress-import/pull/100/files#r761904862), and testing only for:

```suggestion
class TestProvidedShortcodeIsTopLevel(TestCase):
    """
    Block Shortcode Handler has a top_level_property that is True by default.
    It can be set to False to indicate that the shortcode is not a top level HTML tag.
    """

    def test_is_top_level_html_tag_defaults_to_true(self):
        class FooHandler(BlockShortcodeHandler):
            pass

        foo = FooHandler()
        self.assertTrue(foo.is_top_level_html_tag)

    def test_is_top_level_html_tag_can_be_overridden(self):
        class FooHandler(BlockShortcodeHandler):
            is_top_level_html_tag = False

        foo = FooHandler()
        self.assertFalse(foo.is_top_level_html_tag)
```